packages/cozy-harvest-lib/src/components/NewAccountModal.jsx

```diff
@@ -34,7 +36,7 @@ const NewAccountModal = ({ konnector, onSuccess, onDismiss }) => {
   } = useMaintenanceStatus(client, konnector)
   const isMaintenanceLoaded =
     fetchStatus === 'loaded' || fetchStatus === 'failed'
-
+  const isLegacy = !(konnector.oauth || konnector.clientSide)
```

Comment:
this is not legacy, this is `serverSideKonnector`

Reply:
Updated